locales/en-US/notes.properties

```diff
@@ -6,7 +6,7 @@ emptyPlaceHolder=Take a note…
 openingLogin=Opening Login…
 forgetEmail=Forget this Email
 
-syncNotReady=Sorry, Sync is not ready yet, but we will count your click as a vote to hurry it up!
+syncNotReady=Sorry, syncing notes isn’t ready. We will count your click as a vote to hurry it up!
```

Comment:
You need to rebase this, and at this point drop the localization note. It should be OK to change without changing the string ID, since the meaning is the same, it's just a minor nit.